source/coordinates.md

```diff
@@ -407,3 +407,25 @@ the {math}`Y`-axis as it does by default).
 For {math}`\theta_t = \pm\pi/2`, the bend axis is the {math}`x`-axis. 
 A value of {math}`\theta_t = +\pi/2` represents a downward pointing bend.
 
+%---------------------------------------------------------------------------------------------------
+(s:lab.body.transform)=
+### Transformation Between Branch and Element Body Coordinates
+
+The `element body` coordinates are the coordinate system attached to an element. Without any
```

Comment:
Here the phrase "coordinate system" is singular while the word "coordinates" is plural.  We could make these agree by using "... coordinates are the coordinates attached to an element" or "...coordinates define the coordinate system attached to an element".

Reply:
Fixed.